app/screens/channel_info/extra/extra.tsx

```diff
@@ -81,6 +104,79 @@ const Extra = ({channelId, createdAt, createdBy, customStatus, header, isCustomS
         ),
     }), [createdAt, createdBy, theme]);
 
+    const onCopy = useCallback(async (text: string, isLink?: boolean) => {
+        Clipboard.setString(text);
+        await dismissBottomSheet();
+        if ((Platform.OS === OS_VERSION.ANDROID && Number(Platform.Version) < ANDROID_33) || Platform.OS === OS_VERSION.IOS) {
+            showSnackBar({
+                barType: isLink ? SNACK_BAR_TYPE.LINK_COPIED : SNACK_BAR_TYPE.TEXT_COPIED,
+            });
+        }
+    }, []);
```

Comment:
I may be missing something, but this function does not require anything from within the component, right? If that is so, we can move it out of the component and avoid creating it on every render.

Reply:
yeah, you're right. we can write it outside component